filer/static/filer/js/addons/dropzone.init.js

```diff
@@ -0,0 +1,9 @@
+// #DROPZONE#
```

Comment:
Build will be failed, because you don't use 'use strict' and /* global Dropzone */